dandi/bids_validator_xs.py

```diff
@@ -35,6 +35,9 @@ def _get_paths(bids_paths):
         "/.dandi",
         "/.datalad",
         "/.git",
+        "\\.dandi",
+        "\\.datalad",
+        "\\.git",
```

Comment:
why or what for, would ever `os.walk` return names with escaped `.`s?

Reply:
The dots aren't escaped here.  Two backslashes in a non-raw string equals a backslash, which is the Windows path separator.

Reply:
Doh, right.. is there a function to normalize one way or another on windows? You know my allergy to duplication ;-)

Reply:
If you mean always getting the same path separator regardless of OS, no.  I think the better way would be to use `pathlib` instead of string manipulations, both here and throughout the BIDS code, but that might be big enough for a separate PR.

Reply:
ok, let's make them then
```
fr"{os.sep}.dandi",
```
 or just add that `os.sep` while using these in the loop below.